libnetwork/drivers/bridge/bridge_linux_test.go

```diff
@@ -194,17 +197,21 @@ func compareBindings(a, b []types.PortBinding) bool {
 	return true
 }
 
+var a, _ = defaultipam.NewAllocator(ipamutils.GetLocalScopeDefaultNetworks(), nil)
```

Comment:
Why is this allocator package-scoped? It doesn't look like `getIPv4Data` used to have any (local) state which carried between tests, so why change it? None of the allocations are released; is there not a risk of running out?